modules/exploits/windows/http/oats_weblogic_console.rb

```diff
@@ -0,0 +1,388 @@
+##
+# This module requires Metasploit: https://metasploit.com/download
+# Current source: https://github.com/rapid7/metasploit-framework
+##
+
+class MetasploitModule < Msf::Exploit::Remote
+  Rank = ExcellentRanking
+
+  include Msf::Exploit::Remote::HttpClient
+  include Msf::Auxiliary::Report
+
+  def initialize(info={})
+    super(update_info(info,
+      'Name'           => 'Oracle Application Testing Suite WebLogic Server Administration Console War Deployment',
+      'Description'    => %q{
+        This module abuses a feature in WebLogic Server's Administration Console to install
+        a malicious Java application in order to gain remote code execution. Authentication
+        is required, however by default, Oracle ships with a "oats" account that you could
+        log in with, which grants you administrator access.
```

Comment:
It may be useful to mention the `auxiliary/gather/oats_downloadservlet_traversal` module here.

Reply:
Yeah it's mentioned in notes.